Mathlib/Geometry/Manifold/MFDeriv/Atlas.lean

```diff
@@ -349,4 +350,32 @@ lemma isInvertible_mfderiv_extChartAt {y : M} (hy : y ∈ (extChartAt I x).sourc
   have : (extChartAt I x).symm ((extChartAt I x) y) = y := (extChartAt I x).left_inv hy
   rwa [this] at Z
 
+/-- The trivialization of the tangent bundle at a point is the manifold derivative of the
+extended chart.
+Use with care as this abuses the defeq `TangentSpace 𝓘(𝕜, E) y = E` for `y : E`. -/
```

Comment:
Notation question: you use `y` in this doc-string, but `b` in the statement (and also `b` in continuousLinearMapAt_trivializationAt_eq_core): can you ad least make this lemma's name match? (Perhaps changing continuousLinearMapAt_trivializationAt_eq_core would also be helpful.)